docs/configuration.md

```diff
@@ -169,5 +169,108 @@ php artisan up
 ```
 > {tip} You may customize the default maintenance mode template by defining your own template at `resources/views/errors/503.blade.php`.
 
+## Configuration options
+
+|Name|Description|Default|
+|---|---|---|
+|`APP_NAME`|The gallery name|`Lychee`|
+|`APP_ENV`|Environment of your gallery, `production` or `development`|`production`|
+|`APP_DEBUG`|Enable debug mode|`false`|
+|`APP_URL`|The URL of your gallery|`http://localhost`|
+|`APP_KEY`|Your app key, a 32 char string (set during installation)|`null`|
```

Comment:
```suggestion
|`APP_KEY`|Your app key, a 32 char string (set during installation) which is used for encryption|`null`|
```
?

Reply:
```suggestion
|`APP_KEY`|Your app key which is used for encryption (set during installation)|`null`|
```

I'm fine with the addition, but I would leave out the 32 char string. On my installation it is a base 64-encoded random value with 48 characters. I assume the information about 32 characters is outdated.

Reply:
In that case.... *goes elsewhere...*